src/status_im2/contexts/wallet/account/tabs/view.cljs

```diff
@@ -6,22 +6,23 @@
     [status-im2.contexts.wallet.common.activity-tab.view :as activity]
     [status-im2.contexts.wallet.common.collectibles-tab.view :as collectibles]
     [status-im2.contexts.wallet.common.empty-tab.view :as empty-tab]
-    [status-im2.contexts.wallet.common.temp :as temp]
     [status-im2.contexts.wallet.common.token-value.view :as token-value]
-    [utils.i18n :as i18n]))
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
 
 (defn view
-  [{:keys [selected-tab]}]
-  (case selected-tab
-    :assets       [rn/flat-list
-                   {:render-fn               token-value/view
-                    :data                    temp/tokens
-                    :content-container-style {:padding-horizontal 8}}]
-    :collectibles [collectibles/view]
-    :activity     [activity/view]
-    :permissions  [empty-tab/view
-                   {:title        (i18n/label :t/no-permissions)
-                    :description  (i18n/label :t/no-collectibles-description)
-                    :placeholder? true}]
-    :dapps        [dapps/view]
-    [about/view]))
+  [{:keys [selected-tab account-address]}]
+  (let [tokens (rf/sub [:wallet/token-values account-address])]
```

Comment:
nit: somewhat of a chicken and egg situation but imo we should eventually re use the sub @smohamedjavid adds here:
`:wallet/current-viewing-account`
https://github.com/status-im/status-mobile/pull/17798/files#diff-e20c03e779b8a1f282db5d4dc6a02afacf455cf5a98f2f567f7b528728f646cd

we could also update the sub e.g `:wallet/current-viewing-account-token-values` - the name doesn't need to be so verbose but just so the point is clear :)